### PR TITLE
Merge to master PR

### PR DESCRIPTION
**Pull request by wathigo**
Description of tasks
1. Creation of user models
1. Implementation of sign-in/sign-up actions.
2. Implementation of sign-in/sign-up tests
4. Creation of Event models
5. Implementation of host/attend/leave actions.
6. Tests implementation for host/attend/leave actions.

Please review our work. Thank you

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  # default_scope { order(date: :asc) }
+  scope :past_events, lambda {
```
Comment (mcrd25) [inline app/models/event.rb]: Consider using `Time.zone.now` that would give a more accurate reading I think 🤔 
[OPTIONAL]

**Comment by shjang7** [comment]
1. Git flow
Thanks for your advice. Since we didn't see any related project after this, we have made a development branch only. But we would consider it for scaled project for future.

2. Rspec
Thanks for suggestion. It was not a requirement, but it is good to know using for our next project or real world projects. Thanks for sharing the study material!

3. Linter
Yes, we forgot to use it. We may need to set it for this project and next project.

4. Methods in helper path
Sure, we would consider to move it appropriate place. Thank you for sharing the useful link.

5. Comment it out 
Yes, we will remove it.

6. Optional
Thanks for your suggestion. I will discuss with my partner about it.

Thanks for your detail advices, after updating it, we will upload it.

**Comment by wathigo** [comment]
Hello @mcrd25 . We have done the following regarding to the feedback we received from you:
1. Moved helper methods that are not used in the views to the application controller.
2. Removed commented code.
3. Increased the font size of the upcoming and past events.
Please review the changes. Thank you.

**Comment by azdrenymeri** [comment]
### Hello @wathigo @shjang7 
#### Nice job on this one :+1: 
### No further requests from my side, you can ho ahead and merge it.

**Comment by wathigo** [comment]
Thank you @Azdren

**Pull request merged**